test/features/deploy_kogito_build.feature

```diff
@@ -0,0 +1,70 @@
+# Disabled until CLI (used in PR check) supports KogitoRuntime and KogitoBuild - KOGITO-2064
+@disabled
+Feature: Deploy Kogito Build
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed
+
+  Scenario Outline: Build <runtime> remote S2I with native <native> using KogitoBuild
+    When Build <runtime> example service "<example-service>" with configuration:
+      | config | native | <native> |
+
+    Then Build "<example-service>-builder" is complete after <minutes> minutes
+    And Build "<example-service>" is complete after 5 minutes
```

Comment:
Do we need both or only with "And Build "<example-service>" is complete after 5 minutes" is enough? Also, I guess we should use the minutes here: <minutes>

Reply:
Well, I am checking here that both builds are executed.
It depends on what we actually want to check here.

Do we want to make sure that operator creates both builds and executes them successfully? Or we consider it an implementation detail and are interested just in running instance?

I am inclined to check the builds too.

About the minutes - the second build is moreless same for all scenario options, so I kept there hardcoded value. The first build is specific for each build, so I keep the timeout flexible there.

Reply:
Ok